cli/multisig.go

```diff
@@ -266,8 +266,19 @@ var msigInspectCmd = &cli.Command{
 		}
 		fmt.Fprintf(cctx.App.Writer, "Threshold: %d / %d\n", threshold, len(signers))
 		fmt.Fprintln(cctx.App.Writer, "Signers:")
+
+		signerTable := tabwriter.NewWriter(cctx.App.Writer, 8, 4, 2, ' ', 0)
+		fmt.Fprintf(signerTable, "ID\tAddress\n")
 		for _, s := range signers {
-			fmt.Fprintf(cctx.App.Writer, "\t%s\n", s)
+			signerActor, err := api.StateAccountKey(ctx, s, types.EmptyTSK)
```

Comment:
This doesn't work in case the signer is another multisig:
```
./lotus msig inspect f050473

Balance: 0 FIL
Spendable: 0 FIL
Threshold: 2 / 2
Signers:
ERROR: failed to get the public key address of the signer:f050470: failed to get account actor state for f050470: unknown actor code bafkqadtgnfwc6mrpnv2wy5djonuwo

```